### PR TITLE
main: fix call via console script

### PR DESCRIPTION
**Pull request by FFY00**

### CODE REVIEW

**Review by gaborbernat — CHANGES_REQUESTED** [review]
No idea, but you'll need to find out before we can review and merge this.

**Comment by FFY00** [comment]
I should put the tests in a different PR then. We should merge the fix.

**Comment by gaborbernat** [comment]
> I should put the tests in a different PR then. We should merge the fix.

Without a test? I'm not convinced personally.

**Comment by FFY00** [comment]
Yes? This is a really simple stupid issue.

https://github.com/FFY00/python-build/issues/76

I agree that we definitely should have a test, but IMO there's no reason to block the fix because of the test. If this was a harder fix, I would agree, because we want to make sure the fix is correct.

**Comment by gaborbernat** [comment]
We don't know that this works on pypy without the test working 🤷

**Comment by FFY00** [comment]
@pfmoore could you have a look at why the test might be failing on PyPy Windows if you have time? Thanks!

**Comment by pfmoore** [comment]
I've no experience with pypy, but I ran the test on my PC, and it worked fine:

```
>nox -s test-pypy3 -- --run-integration
nox > Running session test-pypy3
nox > Re-using existing virtual environment at .nox\test-pypy3.
nox > pip install .[test]
nox > pytest --cov --cov-config setup.cfg --cov-report=html:D:\Work\Projects\python-build\.nox\test-pypy3\htmlcov --cov-report=xml:D:\Work\Projects\python-build\.nox\test-pypy3\coverage-pypy3.xml tests/ --run-integration
================================================= test session starts =================================================
platform win32 -- Python 3.6.9[pypy-7.3.1-final], pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: D:\Work\Projects\python-build, configfile: setup.cfg
plugins: cov-2.10.1, mock-3.3.1
collected 43 items

tests\test_env.py ...                                                                                            [  6%]
tests\test_integration.py .........                                                                              [ 27%]
tests\test_main.py ............                                                                                  [ 55%]
tests\test_module.py .                                                                                           [ 58%]
tests\test_projectbuilder.py ..................                                                                  [100%]

----------- coverage: platform win32, python 3.6.9-final-0 -----------
Coverage HTML written to dir D:\Work\Projects\python-build\.nox\test-pypy3\htmlcov
Coverage XML written to file D:\Work\Projects\python-build\.nox\test-pypy3\coverage-pypy3.xml


=========================================== 43 passed in 318.26s (0:05:18) ============================================
nox > Session test-pypy3 was successful.
```

So I'm not sure what could be wrong here.

**Comment by pfmoore** [comment]
Sorry, I tested against master. Let me try again against this PR.

**Comment by pfmoore** [comment]
Nope, doesn't fail locally for me:

```
PS 22:18 [Git: pr_77] {00:00.016} D:\Work\Projects\python-build
>nox -s test-pypy3 -- --run-integration
nox > Running session test-pypy3
nox > Re-using existing virtual environment at .nox\test-pypy3.
nox > pip install .[test]
nox > pytest --cov --cov-config setup.cfg --cov-report=html:D:\Work\Projects\python-build\.nox\test-pypy3\htmlcov --cov-report=xml:D:\Work\Projects\python-build\.nox\test-pypy3\coverage-pypy3.xml tests/ --run-integration
================================================= test session starts =================================================
platform win32 -- Python 3.6.9[pypy-7.3.1-final], pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: D:\Work\Projects\python-build, configfile: setup.cfg
plugins: cov-2.10.1, mock-3.3.1
collected 59 items

tests\test_env.py ...                                                                                            [  5%]
tests\test_integration.py .........................                                                              [ 47%]
tests\test_main.py ............                                                                                  [ 67%]
tests\test_module.py .                                                                                           [ 69%]
tests\test_projectbuilder.py ..................                                                                  [100%]

----------- coverage: platform win32, python 3.6.9-final-0 -----------
Coverage HTML written to dir D:\Work\Projects\python-build\.nox\test-pypy3\htmlcov
Coverage XML written to file D:\Work\Projects\python-build\.nox\test-pypy3\coverage-pypy3.xml


=========================================== 59 passed in 819.78s (0:13:39) ============================================
nox > Session test-pypy3 was successful.
```

I'm not sure what to say - something to do with the CI environment?

**Comment by FFY00** [comment]
Thank you. I have restarted the tests, I had done it before but maybe that changed.

**Comment by pfmoore** [comment]
Looks like that fixed it - I assume the codecov failure is unrelated.

**Comment by FFY00** [comment]
Yes, github decided to create a new badge instead of updating the existing one, that's why it still shows as failing.

**Review by gaborbernat — APPROVED** [review]
